src/test/java/edu/kis/powp/jobs2d/window/command/CommandManagerControllerController.java

```diff
@@ -0,0 +1,69 @@
+package edu.kis.powp.jobs2d.window.command;
+
+import edu.kis.powp.jobs2d.command.CommandImporter;
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.FileOpertor;
+import edu.kis.powp.jobs2d.command.json.JsonCommandImporter;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.events.SelectRunCurrentCommandOptionListener;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.jobs2d.features.FeatureManager;
+import edu.kis.powp.observer.Subscriber;
+
+import java.io.IOException;
+import java.util.List;
+
+public class CommandManagerControllerController implements ICommandManagerController {
+
+	private DriverCommandManager commandManager;
+	private CommandImporter importer = new JsonCommandImporter();
+	private CommandsFeature commandsFeature = FeatureManager.getFeature(CommandsFeature.class);
+	private DriverFeature driverFeature = FeatureManager.getFeature(DriverFeature.class);
+
+	public CommandManagerControllerController(DriverCommandManager commandManager) {
+		this.commandManager = commandManager;
+	}
+
+	@Override
+	public void importCommand(String path) throws IOException {
+		String fileContent = FileOpertor.loadFileContent(path);
```

Comment:
I would prefer to put the file manipulation routine on the side of the GUI (file dialogs are already there so this wouldn't ruin SRP more). Please set fileContent as a parameter. The same issue apply to export.